pgspecial/main.py

```diff
@@ -234,6 +234,23 @@ def parse_special_command(sql):
     return (command, verbose, arg.strip())
 
 
+def show_extra_help_command(command, syntax, description):
+    """
+    A decorator used internally for registering help for a command that is not
+    automatically executed via PGSpecial.execute, but invoked manually by the
+    caller (e.g. \watch).
+    """
+
+    @special_command(command, syntax, description, arg_type=NO_QUERY)
+    def placeholder():
+        raise RuntimeError
+
+    def wrapper(wrapped):
+        return wrapped
+
+    return wrapper
```

Comment:
I pattern-matched off of other documentation that was added in a similar way... not sure if it's completely analogous.

It seems odd to me that this documentation is only relevant if the caller (i.e. pgcli) actually invokes `get_watch_command` and yet I've implemented it uncondtionally. 

It would be more correct if the called indicated that it's opted into watching. I briefly looked at making PGSpecial either take in an `include_watch` that conditionally adds the documentation, or having invocation of `get_watch_command` automatically enable it in the documentation... but these felt a bit messy, and is maybe surfacing an underlying interface problem between pgspecial and pgcli.

Reply:
I see no harm in including documentation for a command that is not potentially called. This is fine. We know the 80/20 rule: 80% of the time users will only utilize 20% of the features of the product. Completely normal!